prow/cluster/starter.yaml

```diff
@@ -226,6 +226,7 @@ spec:
       - name: deck
         image: gcr.io/k8s-prow/deck:v20181019-08e9d55c9
         args:
+        - --tide-url=http://tide/
```

Comment:
Come to think of it, do we also need a `- --oauth-url=/github-login` here for the PR status dashboard?

Reply:
Hopefully it degrades gracefully if we don't have one. Do you know @ibzib ?

Reply:
It seems to do the usual thing for Deck, which is spam the logs with error messages while displaying nothing to the user.

Reply:
@Katharine would you know what needs to be done here to enable the PR status dashboard?

If it's any helpful to debug, I've got a config similiar to starter.yaml for an external prow and the PR dashboard doesn't handle the error gracefully and is shown as being stuck at "loading": https://prow.loodse.com/pr

Reply:
The PR dashboard setup is... weird. To make it work, you must:

* Specify some non-empty value for `--oauth-url` (it doesn't actually matter what; it's unused)
* Create a GitHub OAuth app
* Have a relevant yaml config file at `/etc/cookie/secret` (overridden by `--github-oauth-config-file`), containing:
  * `client_id`: the github client ID
  * `client_secret`: the github client secret
  * `redirect_url`: the redirect URL, which in your case is `"https://prow.loodse.com/github-login/redirect"`
  * `scopes`: a list of scopes you need, which I think you can just omit entirely because you don't actually need any.
  * `final_redirect_url`: `"https://prow.loodse.com/pr"` in your case
* Have some base64-encoded random bytes at `/etc/cookie/secret` (path can be overridden by `--cookie-secret`)

(Also, as @BenTheElder says, we should fix it to just not show the menu option if it's not going to work.)

Reply:
So we should hold off on this PR until we fix the menu option, correct? Do we have an issue for tracking this?

Reply:
No, this PR can still go through. It is just configuring Deck to work with Tide. The `- --oauth-url=/github-login` part is what needs to wait.

Reply:
For the pages-not-working issue, it's entirely independent of this PR: you already see a PR dashboard (that doesn't work) in a default configuration. You also currently see a Tide dashboard (that also doesn't work). Presumably this PR will have better odds of the tide dashboard actually working.

Reply:
Made #9996 to track this.